cdap-watchdog/src/main/java/co/cask/cdap/logging/write/FileMetaDataManager.java

```diff
@@ -264,6 +271,13 @@ public String call() throws Exception {
                   LOG.debug("Got file {} with start time {}", file, Bytes.toLong(colName));
                 }
 
+                if (Strings.isNullOrEmpty(namespacedLogDir)) {
+                  LOG.warn("Delete metadata for {} namespace", namespaceId.getEntityName());
```

Comment:
include the file name in the message? (Because the other similar messages here also include it).